tests/kubeless/kubeless-tests.go

```diff
@@ -4,12 +4,14 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"math/rand"
 	"net/http"
 	"os/exec"
 	"strings"
 	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/avast/retry-go"
```

Comment:
We could avoid that new dependency by using `k8s.io/apimachinery/pkg/util/wait` instead.

Reply:
true, i just use it as lots of other kyma components also make use of this library